web/controllers/post_controller.ex

```diff
@@ -8,6 +8,23 @@ defmodule Lift.PostController do
 
   plug Guardian.Plug.EnsureAuthenticated, handler: Lift.TokenController
 
+  def popular(conn, %{"offset" => offset, "page" => page} = params, user, _claims) do
+    radius =
+      with {o, _} <- Integer.parse(offset || "5"),
+           {p, _} <- Integer.parse(page   || "0"),
+       do: (p * o)..(((p + 1) * o) - 1)
```

Comment:
Eh, use schrivener for pagination instead of this? See https://github.com/Lift-app/lift-api/blob/master/web/controllers/post_controller.ex#L18